data-raw/make_metadata_eml.R

```diff
@@ -0,0 +1,60 @@
+library(EMLaide)
```

Comment:
should this file be in this repo?

Reply:
I got rid of this file. I was using it to test functions when refactoring but it does not need to be here anymore.